resources/site-packages/xbmctorrent/scrapers/rutor.py

```diff
@@ -10,7 +10,7 @@
 from xbmctorrent.utils import ensure_fanart
 
 BASE_URL = "%s/" % plugin.get_setting("base_rutor")
-DOWNLOAD_URL = "http://d.rutor.org/"
+DOWNLOAD_URL = BASE_URL + "parse/d.rutor.org/"
```

Comment:
Tested with new-rutor.org and should work with others. Mirrors using different link format {mirror}/parse/d.rutor.org/{torrent Id}